### DISCUSSION TITLE
Blocked out of my account!!!!!

### DISCUSSION THREAD

**Discussion by omerhac** [vercel/community#6951] [Account Issue]
### Question

I have been blocked out of my team eikona-io!!!!
please help !!!!!!!!

**Comment by omerhac** [comment]
still!!!!
your service is the worst. there is no way to contact you at all.
Im deleting my account as soon as this is fixed.

**Comment by amyegan** [comment]
Thanks for reaching out, @omerhac. I'm not able to find a team with that name or an account associated with your GitHub username. You may already have gotten help from our Customer Success team. But if you're still having trouble, please let me know the Vercel username or ID and I would be happy to help you with this.
